.github/workflows/publish-docker-images.yml

```diff
@@ -79,7 +64,6 @@ jobs:
         id: tagprep_cli
         run: |
           DOCKER_TAGS="${ZROK_CONTAINER_IMAGE_REPO}:${ZROK_CONTAINER_IMAGE_TAG}"
-          DOCKER_TAGS+=",${ZROK_CONTAINER_IMAGE_REPO}:latest"
```

Comment:
stop publishing container images as `:latest` because now we're calling this workflow during the pre-release cycle, and have a separate workflow for promoting to latest/stable when the GitHub release is marked stable.